src/editor/plugins/lsp/copilot/index.ts

```diff
@@ -30,7 +30,7 @@ import {
   posToOffset,
 } from '@kittycad/codemirror-lsp-client'
 import { reportRejection } from '@src/lib/trap'
-import { deferExecution } from '@src/lib/utils'
+import { deferredCallback } from '@src/lib/utils'
```

Comment:
This name really confused me while I was working in here, so I think renaming it was worth the noise. It has nothing to do with _engine_ execution, just "executing" a function, so I want to make that more clear.